BrowserKit/Sources/ComponentLibrary/Buttons/PrimaryRoundedButton.swift

```diff
@@ -32,21 +32,22 @@ public class PrimaryRoundedButton: LegacyResizableButton, ThemeApplicable {
     override init(frame: CGRect) {
         super.init(frame: frame)
 
-        titleLabel?.font = DefaultDynamicFontHelper.preferredBoldFont(
+        configuration?.setFont(DefaultDynamicFontHelper.preferredBoldFont(
```

Comment:
The font should be set in the configure method.

Reply:
Done!